Add-On UI/javascript/submit_form.html

```diff
@@ -0,0 +1,248 @@
+<!-- Javascript Functions for extracting data filled by user and 
+    calling server-side Apps Script function to get the result -->
+
+<script>
+
+  var form = document.getElementById("form");  //fetching form
+  var done = document.getElementById("done");  //fetching the submit button
+  
+  // Function called when user clicks the Compute (submit) button
+  function submitForm(intentName,event) {
+  
+    event.preventDefault();
```

Comment:
add a comment on what the line accomplishes. Also add what is the default action that you are cancelling , and why if it is not too complex.

Reply:
Okay, I will add a short explanation